opentelemetry-api/src/opentelemetry/resources/__init__.py

```diff
@@ -11,3 +11,56 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from typing import Any, Dict, Union
+
+
+class Resource:
+    def __init__(self, labels: Dict[str, str]):
+        """
```

Comment:
Do we want
```
"""foo
bar
baz
```
or
```
"""
foo
bar
baz
"""
```

Either way looks good to me as long as we're consistent (e.g. https://github.com/open-telemetry/opentelemetry-python/blob/master/opentelemetry-api/src/opentelemetry/trace/__init__.py#L74).

Reply:
The first one please, like https://github.com/open-telemetry/opentelemetry-python/pull/61#pullrequestreview-266303416.

Reply:
adjusted to use the sphinx-style docstrings as suggested.